### DISCUSSION TITLE
Regions: Text annotation jumping around if regions overlap

### DISCUSSION THREAD

**Discussion by stcoats** [katspaugh/wavesurfer.js#3087] [Q&A] [answered]
Hello! 

Thanks again for this great library. I have a question -- feel free to delete this post if you feel it is too simple or not useful. 

I would like to have two annotation tiers for a single waveform. I have done it like this: 

https://jsfiddle.net/105auhrc/

The issue is that when the regions overlap, the vertical position of the text moves. It only stays at the top of the region if there is no overlap. The effect is more pronounced if there are many overlapping regions. 

I understand this is related to CSS. Could anyone give me a tip on how to get the annotations to stay at the top of the corresponding region?

Thanks for your help!

**Comment by stcoats** [comment, marked as answer]
I fixed it with 

```
#waveform ::part(region-content) {
    position: absolute;
    top: 0;
    margin-top: 0px;
}
```

**Reply by katspaugh** [reply]
Cool, glad you figured it out! I was going to suggest something similar as well.
